src/Analyzers/Core/CodeFixes/UseSystemHashCode/UseSystemHashCodeCodeFixProvider.cs

```diff
@@ -82,7 +82,7 @@ public override Task RegisterCodeFixesAsync(CodeFixContext context)
 
                     // Only if there was a base.GetHashCode() do we pass in the ContainingType
                     // so that we generate the same.
-                    var containingType = accessesBase ? method.ContainingType : null;
+                    var containingType = accessesBase ? method!.ContainingType : null;
```

Comment:
GetREquiredDeclaredSymbol?  or GetDeclaredSymbol??throw?

Reply:
`GetHashedMembers` allows null, but you can't enter this `if` statement in that case.